ena-submission/src/ena_deposition/ena_submission_helper.py

```diff
@@ -380,57 +383,9 @@ def create_flatfile(
         with tempfile.NamedTemporaryFile(delete=False, suffix=".embl") as temp:
             filename = temp.name
 
-    seqIO_moleculetype = {
-        MoleculeType.GENOMIC_DNA: "DNA",
-        MoleculeType.GENOMIC_RNA: "RNA",
-        MoleculeType.VIRAL_CRNA: "cRNA",
-    }
-
-    embl_content = []
-
-    multi_segment = set(unaligned_nucleotide_sequences.keys()) != {"main"}
-
-    for seq_name, sequence_str in unaligned_nucleotide_sequences.items():
-        if not sequence_str:
-            continue
-        reference = Reference()
-        reference.authors = authors
-        sequence = SeqRecord(
-            Seq(sequence_str),
-            id=f"{accession}_{seq_name}" if multi_segment else accession,
-            annotations={
-                "molecule_type": seqIO_moleculetype[moleculetype],
-                "organism": organism,
-                "topology": organism_metadata.get("topology", "linear"),
-                "references": [reference],
-            },
-            description=description,
-        )
-
-        source_feature = SeqFeature(
-            FeatureLocation(start=0, end=len(sequence.seq)),
-            type="source",
-            qualifiers={
-                "molecule_type": str(moleculetype),
-                "organism": organism,
-                "country": country,
-                "collection_date": collection_date,
-            },
-        )
-        sequence.features.append(source_feature)
-
-        with tempfile.NamedTemporaryFile(delete=False, suffix=".embl") as temp_seq_file:
-            SeqIO.write(sequence, temp_seq_file.name, "embl")
-
-        with open(temp_seq_file.name, encoding="utf-8") as temp_seq_file:
-            embl_content.append(temp_seq_file.read())
-
-    final_content = "\n".join(embl_content)
-
     gzip_filename = filename + ".gz"
 
-    with gzip.open(gzip_filename, "wt", encoding="utf-8") as file:
```

Comment:
the simple gzip function I had before is all you need (we anyways need to write the files to disk for the webin-cli to upload to ena)